cmd/storj-sim/network.go

```diff
@@ -217,12 +229,21 @@ func newNetwork(flags *Flags) (*Processes, error) {
 	versioncontrol.ExecBefore["run"] = func(process *Process) error {
 		return readConfigString(&versioncontrol.Address, versioncontrol.Directory, "address")
 	}
-
 	// gateway must wait for the versioncontrol to start up
 
+	// start a test redis server
+	rootpath := flags.Redis
+	if flags.Redis == "testServer" {
```

Comment:
why not instead check for empty string?

Reply:
empty string could also indicate we want to use bolt... unless we want to eliminate that option

Reply:
I think we can stop worrying about bolt.